impl/src/main/java/org/jboss/weld/logging/BeanLogger.java

```diff
@@ -505,4 +507,7 @@
 
     @Message(id = 1572, value = "Cannot create instance of session bean from Annotated Type {0} before AfterDeploymentValidation phase.", format = Format.MESSAGE_FORMAT)
     CreationException initABDnotInvoked(Object bean);
+
+    @Message(id = 1573, value = "Not a business method {0}. Do not call non-public methods on EJB's.", format = Format.MESSAGE_FORMAT)
+    EJBException nonPublicMethodInvoked(Object method);
```

Comment:
Note that EJB API is an optional dependency and `BeanLogger` is used in SE and other environments. So I think we should not import this class here.

Reply:
Agreed, moving this to `EnterpriseBeanProxyMethodHandler`